sources/packages/backend/libs/integrations/src/sfas-integration/sfas-files/sfas-individual-record.ts

```diff
@@ -12,27 +12,27 @@ export class SFASIndividualRecord extends SFASRecordIdentification {
    * SFAS Unique id.
    */
   get id(): number {
-    return +this.line.substr(3, 10);
+    return +this.line.substring(3, 13);
   }
```

Comment:
I'm thinking maybe we can have a e2e (unit) test to verify that the an SFAS record is correctly saved to an instance since there are quite a few changes here. Please let me know your thoughts.

Reply:
I see your point and I think it's valid but I won't add any tests for now. We may check if there is already a ticket for that, otherwise I can create a ticket to test it.